### PR TITLE
move wiki to repo

### PR DESCRIPTION
**Pull request by shiyuhang0**
### What problem does this PR solve? 
move some doc from wiki to repo
- configrutions
- stale_read feature

The docs which may be changed by contributors need to be put in the repo, others should be put in the wiki

### CODE REVIEW

**Comment by shiyuhang0** [comment]
@zhangyangyu @xuanyu66 @birdstorm PTAL

File: README.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ Most of the TiSpark logic is inside a thin layer, namely, the [tikv-client](http
 - [Getting Started](https://github.com/pingcap/tispark/wiki/Getting-Started)
 - [User Guide](https://github.com/pingcap/tispark/blob/master/docs/userguide.md)
 - [Authorization and authentication](./docs/authorization_userguide.md)
-- [Configuration](https://github.com/pingcap/tispark/wiki/Configuration)
+- [Configuration](./docs/configuration.md)
```
Comment (zhangyangyu) [inline README.md:28]: No stale read ToC entry, delibrately?

File: docs/stale_read.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+TiSpark support stale read with a new configuration `spark.tispark.stale_read`
+- The configurations accept timestamp (ms) which is Long type
+- The configuration better meet this condition `${now} - ${spark.tispark.stale_read} + ${sql execution time}) < ${GC lifetime}`, or you may get unexception data
+
+## Limitation
+- We suggest you don't use DML with stale read, for they need current schema rather than historical schema
+- Stale read is not thread safety in a SparkSession, but you can execute SELECT statement concurrently in different SparkSession
+
+
+
```
Comment (zhangyangyu) [inline docs/stale_read.md:20]: Reformat plz. Keep style(like spaces, empty lines between sections) consistent.

File: README.md (hunk 1 above)
Comment (shiyuhang0) [reply README.md:28]: stale read's link put in the configuration.md. After all, a TOC per feature may too messy

File: docs/stale_read.md (hunk 2 above)
Comment (zhangyangyu) [reply docs/stale_read.md]: LGTM

Comment (shiyuhang0) [reply docs/stale_read.md]: done

**Review by xuanyu66 — APPROVED** [review]

**Review by zhangyangyu — APPROVED** [review]

**Review by sunxiaoguang — APPROVED** [review]

**Pull request merged**